hil/cli/node.py

```diff
@@ -194,13 +253,19 @@ def node_console():
 @node_console.command(name='show', short_help='Show console')
 @click.argument('node')
 def node_show_console(node):
-    """Display console log for <node>
+    """Display console log for <node>"""
```

Comment:
not sure what happened here, but this isn't a part of this PR at all (and in fact, this reverts part of the cli).